oshdb-util/src/main/java/org/heigit/ohsome/oshdb/util/tagtranslator/TagTranslator.java

```diff
@@ -72,33 +78,30 @@ public TagTranslator(Connection conn) throws OSHDBKeytablesNotFoundException {
 
     // test connection for presence of actual "keytables" tables
     EnumSet<TableNames> keyTables =
-        EnumSet.of(TableNames.E_KEY, TableNames.E_KEYVALUE, TableNames.E_ROLE);
+        EnumSet.of(E_KEY, E_KEYVALUE, E_ROLE);
     for (TableNames table : keyTables) {
       try (Statement testTablePresentQuery = this.conn.createStatement()) {
-        testTablePresentQuery.execute("select 1 from " + table.toString() + " LIMIT 1");
+        var select = format("select 1 from %s limit 1", table);
+        testTablePresentQuery.execute(select);
```

Comment:
In `OSHDBH2.java` you named all variables containing a String typed SQL statement `…Sql`. If you want to do something like that in general, you could use a name like this here:
```suggestion
        var selectSql = format("select 1 from %s limit 1", table);
        testTablePresentQuery.execute(selectSql);
```